### ISSUE TITLE
Accessibility: implement unit and UI tests for accessible navigation

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#3285]
* .NET Core Version: 

.NET SDK (reflecting any global.json):
 Version:   5.0.100-preview.5.20260.4
 Commit:    0950d416f9

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.18363
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\5.0.100-preview.5.20260.4\

Host (useful for support):
  Version: 5.0.0-preview.5.20260.5
  Commit:  09ee4814cb

* Have you experienced this same bug with .NET Framework?: 

**Problem description:**

There is no covering by unit tests for accessibility navigation including `IAccessible.accNavigate` and 
`IRawElementProviderFragment.Navigate` method implementations.

**Expected behavior:**
There should be test covering for accessible navigation so if navigation is broken by some changes CI will notify about it.
- Need to have basic unit tests for unit functionality.
- Need to have per-control scenarios for controls' accessible navigation.

**Minimal repro:**
- Change the implementation of accessible navigation for some control, for instance add return null for some control's accessible object `FragmentNavigate` and notify that Inspect shows empty accessible tree for the this control in inspect. Please note that there is no tests for this.

**Comment by SergeySmirnov-Akvelon** [comment]
To address this problem, a #6094 was created with a list of test classes to which missing unit tests should be added
